### ISSUE TITLE
autotools-non-release-build ?

### ISSUE THREAD

**Issue by rderooy** [flathub/flatpak-builder-lint#107]
Can someone tell me what this means?

```
flatpak run --command=flatpak-builder-lint org.flatpak.Builder --exceptions com.dosbox_x.DOSBox-X.yaml
 in dir /srv/buildbot/worker/build-x86_64-5/build (timeout 1200 secs)
 watching logfiles {}
 argv: b'flatpak run --command=flatpak-builder-lint org.flatpak.Builder --exceptions com.dosbox_x.DOSBox-X.yaml'
 using PTY: False
{
    "errors": [
        "module-dosbox-x-autotools-non-release-build"
    ]
}
If you think problems listed above are a false positive, please report it here:
  https://github.com/flathub/flatpak-builder-lint
program finished with exit code 1
elapsedTime=1.308365
```
https://github.com/flathub/com.dosbox_x.DOSBox-X/pull/10

**Comment by rderooy** [comment]
Found the issue by searching through the linter source code. Had to disable the debugger. This means dosbox-x flatpak will no longer have a DOS debugger.
https://github.com/flathub/flatpak-builder-lint/blob/master/flatpak_builder_lint/checks/modules.py

```
elif opt.startswith("--enable-debug") and not opt.endswith("=no"):
                        self.errors.add(f"module-{name}-autotools-non-release-build")
```
dosbox-x uses that flag to enable the debug option for debugging not dosbox-x, but rather dos programs running inside. e.g., this is what you get when debug is enabled:

![image](https://user-images.githubusercontent.com/17701098/209650028-a43b4ca4-f726-43c6-b485-2cb4e188b326.png)

**Comment by barthalion** [comment]
Please submit a PR to https://github.com/flathub/flatpak-builder-lint/blob/master/flatpak_builder_lint/staticfiles/exceptions.json to exclude your app.
